### DISCUSSION TITLE
Listing windows within one condition?

### DISCUSSION THREAD

**Discussion by Noctimor** [WarmUpTill/SceneSwitcher#620] [Q&A] [answered]
Hello! I've loved using Advanced Scene Switcher, it's solved my wish for a HDR whitelist of sorts. I play games in HDR and always output in SDR, but game capture's color space needs to be manually set between SDR and HDR games to look correct. Not anymore! Here's what I've been doing:
![image](https://user-images.githubusercontent.com/13789545/205760999-c05dafb1-d6a7-4b49-847e-aca9221105b4.png)
![image](https://user-images.githubusercontent.com/13789545/205761089-dc00d0f1-6422-4428-9ff6-c140b9217908.png)
This has been working great, but I was wondering if there wasn't a better way for me to be inputting window titles. As it is now, I'm making identical if statements for each window, and every time I go to add a new game it's a longer scroll. It's perfectly functional but had me wondering if there was a different way to handle this within one condition somehow. 

...

Also another thing (I should probably separate into its own post), specifically that Call of Duty HQ window seen in the screenshot, the name of the window for the latest game, needs to be 'reset' at times. I'll launch the game, notice the macro hasn't taken effect, go to edit, open the dropdown for that condition and select the exact same name in the menu, and it suddenly resolves itself. I'm not sure what would cause that, but wonder if it's anticheat related, like rotating through executables or something like that

**Comment by WarmUpTill** [comment, marked as answer]
>This has been working great, but I was wondering if there wasn't a better way for me to be inputting window titles. As it is now, I'm making identical if statements for each window, and every time I go to add a new game it's a longer scroll. It's perfectly functional but had me wondering if there was a different way to handle this within one condition somehow.

I can't really think of a different way to handle this unfortunately.
Do you have anything in mind?

>Also another thing (I should probably separate into its own post), specifically that Call of Duty HQ window seen in the screenshot, the name of the window for the latest game, needs to be 'reset' at times. I'll launch the game, notice the macro hasn't taken effect, go to edit, open the dropdown for that condition and select the exact same name in the menu, and it suddenly resolves itself. I'm not sure what would cause that, but wonder if it's anticheat related, like rotating through executables or something like that

Just guessing, but maybe something changes about the window title that is not directly visible in the name (e.g. added spaces).
Have you tried using the "Process" condition type?
It might be more reliable for this particular application.

**Reply by Noctimor** [reply]
> Have you tried using the "Process" condition type?

Didn't know this existed! Seems to be working out so far, thanks!
As far as having anything in mind, it's honestly fine as it is, I think my use case is always going to end up being a long list, but one thing would be having Home/End key support for the scroll bar. Right now PgUp/PgDn works, but not jumping to the top/bottom with Home/End. That'd be nice, but even without it's fine! Thanks again for all the work you've put into this!
